org.onionware.onionmedia.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.onionware.onionmedia
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+
+modules:     
+  - name: Desktop
+    buildsystem: simple
+    build-commands:
+      - cp -r . /app
+    sources:
+      - type: archive
+        url: https://github.com/onionware-github/OnionMedia-X/releases/download/2.0.0.0/flatpak-resources.zip
+        sha256: 4b97e11f562db8fbc2be0babed349f477af0ae848faaff5383cbc2459bceaf53
+        dest-filename: flatpak-resources.zip
+        
+  - name: OnionMedia
+    buildsystem: simple
+    build-commands:
+      - cp -r . /app/bin
+      - install -D start.sh /app/bin/start.sh
+      - chmod -R +x /app/bin
+    sources:
+      - type: archive
+        url: https://github.com/onionware-github/OnionMedia-X/releases/download/2.0.0.0/OnionMediaX_2.0.0.0-linux.X64.zip
+        sha256: 0271e87a6d5c5d7044a4eab821fc7a8a5ee7ab37708bef074a001ba3e6ba417e
+        dest-filename: publish.zip
+        
+    cleanup:
+      - '*.pdb'
+      - 'FluentAvalonia.ProgressRing.xml'
+      - 'YoutubeDLSharp.xml'
+        
+finish-args:
+  - --filesystem=host
```

Comment:
this is a very wide filesystem permissions.

Reply:
The user should be free to store converted/downloaded files everywhere.